{"name": "输入已存在的用户名注册失败", "status": "passed", "attachments": [{"name": "stdout", "source": "ee5f1966-3867-421f-ab49-fc046005907e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "Parameter", "value": "{'case_no': '9', 'casename': '输入已存在的用户名注册失败', 'mode': '注册', 'data': 'userAccount=979172251@qq.com', 'assert_way': 'xpath=/html/body/div[2]/span/div/div/div/span', 'result': 'The mailbox is already registered'}"}], "start": 1650519902086, "stop": 1650519906294, "uuid": "c25eaf01-1810-4510-96df-5ae62346579f", "historyId": "f44382204f6c23bd5640584e5a08c4dd", "testCaseId": "91efbca2877ad29cfc72c696a08d59d8", "fullName": "test_main.Test_Main#test_regist", "labels": [{"name": "feature", "value": "注册"}, {"name": "suite", "value": "test_main"}, {"name": "subSuite", "value": "Test_Main"}, {"name": "host", "value": "DESKTOP-NB3FDE0"}, {"name": "thread", "value": "17060-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_main"}]}